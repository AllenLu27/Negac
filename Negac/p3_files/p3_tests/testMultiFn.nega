int fn(){
}

int fn(){
	int a;
	bool a;
}
