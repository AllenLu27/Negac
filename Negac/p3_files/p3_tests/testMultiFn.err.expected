FATAL [4,5]: Multiply declared identifier
FATAL [6,7]: Multiply declared identifier
Semantic analysis failed
